lib/puppet/resource/type_collection.rb

```diff
@@ -187,7 +187,7 @@ def version
       if environment.config_version.nil? || environment.config_version == ""
         @version = Time.now.to_i
       else
-        @version = Puppet::Util::Execution.execute([environment.config_version]).strip
+        @version = Puppet::Util::Execution.execute(environment.config_version).strip
```

Comment:
Today I learned new things about ruby and process execution.

@joshcooper that's good to know. If that's why this and other calls to execute are all still arrayified, that would be good to note in comments and/or the yard doc for the execute method.